test_othermlpackages/test_tensorflow.py

```diff
@@ -9,7 +9,8 @@
 tf = pytest.importorskip("tensorflow")
 from tensorflow.keras.layers import Dense # noqa
 from tensorflow.keras.models import Sequential # noqa
-from tensorflow.keras.wrappers.scikit_learn import KerasClassifier # noqa
+import scikeras
```

Comment:
Is this specific import actually used? `flake8` is still complaining

Reply:
Are you able to see the build reports @pritishaw1 ?

Reply:
Yes @riedgar-ms, the highlighted import was not being used so I have removed that in the updated PR.